cmd/helios-cli/main.go

```diff
@@ -16,46 +16,56 @@
 package main
 
 import (
-	"flag"
-	"fmt"
-	"os"
+    "flag"
+    "fmt"
+    "os"
 
-	"github.com/good-night-oppie/helios-engine/cmd/helios-cli/internal/cli"
+    "github.com/good-night-oppie/helios-engine/cmd/helios-cli/internal/cli"
+)
+
+// Version metadata. Overridden at build time via -ldflags.
+var (
```

Comment:
Nice approach with ldflags-backed vars. Consider adding a Makefile target to inject `-ldflags` for release builds so version/commit/date are always populated.